internal/security/anchore_client.go

```diff
@@ -317,6 +319,72 @@ func (a anchoreClient) AddRegistry(ctx context.Context, registry Registry) error
 	return nil
 }
 
+func (a anchoreClient) GetRegistry(ctx context.Context, registryName string) ([]anchore.RegistryConfiguration, error) {
+	a.logger.Info("getting anchore registry", map[string]interface{}{
+		"registryName": registryName,
+	})
+
+	opts := &anchore.GetRegistryOpts{}
+	registry, resp, err := a.getRestClient().RegistriesApi.GetRegistry(a.authorizedContext(ctx), registryName, opts)
+
+	a.logger.Debug("registry", map[string]interface{}{
+		"response":   resp.Status,
+		"registriey": registry,
+	})
+
+	if err != nil || (resp.StatusCode != http.StatusOK) {
+		return nil, errors.WrapIfWithDetails(err, "failed to get registry", registryName)
+	}
+
+	return registry, nil
+}
+
+func (a anchoreClient) UpdateRegistry(ctx context.Context, registry Registry) error {
+	fnCtx := map[string]interface{}{"registry": registry.Registry}
+	a.logger.Info("updating anchore registry", fnCtx)
+
+	// registryType := registry.Type
```

Comment:
What is the purpose of this commented-out section?

Reply:
The commented-out lines implement the update registry using the `anchore-engine` UpdateRegistry function but unfortunately, this function doesn't work. So the UpdateRegistry had to be workarounded and the I forgot to delete the commented-out lines.

Reply:
fixed